security-analysis/src/test/java/eu/itesla_project/security/SecurityTest.java

```diff
@@ -27,63 +34,144 @@
+                     "contingency1,converge,,,,,,," + System.lineSeparator() +
+                     ",,action2,,,,,," + System.lineSeparator() +
+                     ",,,line1,CURRENT,20',1100.00,1000.0,110" + System.lineSeparator() +
+                     ",,,line2,CURRENT,10',950.000,900.0,106" + System.lineSeparator(),
+                     writer.toString());
     }
 
     @Test
     public void printPostContingencyViolationsWithPreContingencyViolationsFiltering() throws Exception {
         StringWriter writer = new StringWriter();
         try {
-            Security.printPostContingencyViolations(createResult(), writer, formatterFactory, formatterConfig, null, true);
+            Security.printPostContingencyViolations(result, writer, formatterFactory, formatterConfig, null, true);
         } finally {
             writer.close();
         }
-        assertEquals(writer.toString(), "Post-contingency limit violations" + System.lineSeparator() +
-                                               "Contingency,Status,Action,Equipment,Violation type,Violation name,Value,Limit,Charge %" + System.lineSeparator() +
-                                               "contingency1,converge,,,,,,," + System.lineSeparator() +
-                                               ",,,line2,CURRENT,10',950.000,900.0,106" + System.lineSeparator());
+        assertEquals("Post-contingency limit violations" + System.lineSeparator() +
+                     "Contingency,Status,Action,Equipment,Violation type,Violation name,Value,Limit,Charge %" + System.lineSeparator() +
+                     "contingency1,converge,,,,,,," + System.lineSeparator() +
+                     ",,action2,,,,,," + System.lineSeparator() +
+                     ",,,line2,CURRENT,10',950.000,900.0,106" + System.lineSeparator(),
+                     writer.toString());
+    }
+    
+    @Test
+    public void printLimitsViolations() {
+        assertEquals("+---------+--------------+---------------+----------------+----------------+--------+--------+------------------+----------+\n" +
+                     "| Country | Base voltage | Equipment (2) | Violation type | Violation name | value  | limit  | abs(value-limit) | charge % |\n" +
+                     "+---------+--------------+---------------+----------------+----------------+--------+--------+------------------+----------+\n" +
+                     "|         |              | line1         | CURRENT        | 20'            | 1100.0 | 1000.0 | 100.0            | 110      |\n" +
+                     "|         |              | line2         | CURRENT        | 10'            | 950.0  | 900.0  | 50.0             | 106      |\n" +
+                     "+---------+--------------+---------------+----------------+----------------+--------+--------+------------------+----------+",
+                     Security.printLimitsViolations(Arrays.asList(line1Violation, line2Violation), new LimitViolationFilter()));
```

Comment:
I'm not sure this code is windows compliant: use System.lineSeparator instead of "\n". You can also use the modern way to concatenate the lines